packages/deprecation-cop/lib/deprecation-cop-view.js

```diff
@@ -137,7 +137,7 @@ export default class DeprecationCopView {
                   <span className="text-warning icon icon-alert" />
                   <div
                     className="list-item deprecation-message"
-                    innerHTML={marked(deprecation.getMessage())}
+                    dangerouslySetInnerHTML={marked(deprecation.getMessage())}
```

Comment:
This is JSX for [etch](https://github.com/atom/etch) not react so I don't think `dangerouslySetInnerHTML` will work.